internal/net/http/transport/roundtrip.go

```diff
@@ -41,50 +42,66 @@ func NewExpBackoff(opts ...Option) http.RoundTripper {
 	return e
 }
 
-func (e *ert) RoundTrip(req *http.Request) (res *http.Response, err error) {
+// RoundTrip round trip the HTTP request and return the response.
+// If backoff is not set, the default roundTrip implementation will be used.
+// The default roundTrip implementation round trip the request, and return any round trip error occurred.
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg?style=flat-square)
we don't have to mention about the behavior of the default RoundTripper. it may differ if the variable `ert.transport` set.